Sources/Networking/Operations/GetOfferingsOperation.swift

```diff
@@ -44,7 +44,7 @@ private extension GetOfferingsOperation {
             return
         }
 
-        let request = HTTPRequest(method: .get, path: .getOfferings(appUserID: appUserID))
+        let request = HTTPRequest(method: .get, path: .getOfferings(appUserID: appUserID), timeout: httpClient.timeout)
```

Comment:
This feels backwards. `HTTPClient` is already initialized with a timeout, so why pass the very same timeout to every request?

Reply:
You're right, I was being heavy handed with a replacement

Reply:
✅